SokolTextGame/Commands.cs

```diff
@@ -39,7 +41,24 @@ public void Execute(string[] words, World world)
             if (string.Join(" ", words) == "look") Console.Write("Look... What's next? You can look at your weapon\n");
             else if (string.Join(" ", words) == "look at") Console.Write("Look at... what?\n");
             else if (string.Join(" ", words) == "look at my weapon") Console.Write(world?.player?.Weapon.Description);
-            else Console.Write("I know the command \"look at my weapon\"\n");
+            else if (string.Join(" ", words) == "look at guard")
+            {
+                if (string.Join(" ", world.CurrentLocation()?.ObjectOnLocation?.Name) != null &&
```

Comment:
Please do the same thing as in the `talk` command and make `world.CurrentLocation().ObjectOnLocation` a separate variable, and then use it as a variable in all your `if`s. Don't write the full `world.CurrentLocation().ObjectOnLocation` multiple times in the same method.